docs/run-random-beacon.adoc

```diff
@@ -0,0 +1,402 @@
+:toc: macro
+
+= Run Random Beacon
+
+:icons: font
+:numbered:
+toc::[]
+
+== System Considerations
+
```

Comment:
Some sort of intro would be good here. Something like “The Keep network expects certain properties of its running nodes. To provide these properties, a client deployment should meet a few criteria.”